### DISCUSSION TITLE
blog-celular-tocando-na-aula-v2

### DISCUSSION THREAD

**Discussion by giscus** [MrPowerGamerBR/PersonalWebsiteComments#3] [Fan Comments]
# blog-celular-tocando-na-aula-v2

E se reclamar que a professora deixou o celular ligado na aula ganha de brinde um esporro.

https://mrpowergamerbr.com/br/blog/celular-tocando-na-aula-v2/

**Comment by brviictoor** [comment]
```kt
val funny = true
```
